src/wp-includes/block-supports/background.php

```diff
@@ -58,9 +58,9 @@ function wp_render_background_support( $block_content, $block ) {
 		return $block_content;
 	}
 
-	$background_image_source = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundImage', 'source' ), null );
-	$background_image_url    = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundImage', 'url' ), null );
-	$background_size         = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundSize' ), 'cover' );
+	$background_image_source = $block_attributes['style']['background']['backgroundImage']['source'] ?? null;
```

Comment:
Is there ever a possibility that one of the keys will not exist in the structure? 

If yes, then a Warning (PHP 8) | Notice (PHP 7) will happen https://3v4l.org/QaHs5.

Reply:
This question applies to all instances in this PR.

Reply:
Yes, there is a chance that one of the keys will not exist in the structure.
However, no warnings or notices will ever be thrown...

The code in the posted example looks like this:
```php
$source = $block_attributes['style']['background']['backgroundImage']['source']
    ? $block_attributes['style']['background']['backgroundImage']['source']
    : null;
```
That will definitely throw warnings/notices... But the code in this PR is not the same as the one in the example above. Instead, it looks like this:
```php
$source = $block_attributes['style']['background']['backgroundImage']['source'] ?? null;
```
The `??` here is just a short syntax for an `isset()` check, so if we want to be more verbose, the above would be the same as this - just shorter, and IMO easier to follow.
```php
$source = isset( $block_attributes['style']['background']['backgroundImage']['source'] )
    ? $block_attributes['style']['background']['backgroundImage']['source']
    : null;
```
That will not throw any notices or warnings, since `isset()` works fine with nested keys, and will simply return false if one of them is not set.
If we want to be extra verbose we could use a separate `isset()` check for each level, and it would still be the same... So in this case, the `??` syntax would be the same as this:
```php
if ( isset( $block_attributes['style'] )
    && isset( $block_attributes['style']['background'] )
    && isset( $block_attributes['style']['background']['backgroundImage'] )
    && isset( $block_attributes['style']['background']['backgroundImage']['source'] )
) {
    $source = $block_attributes['style']['background']['backgroundImage']['source'];
} else {
    $source = null;
}
```

Reply:
Just a heads-up: no need for four `isset()`, one will do:
```php
if ( isset( $block_attributes['style']['background']['backgroundImage']['source'] )) {
```

...and even if it would be for different variables, one will still do as `isset()` takes multiple parameters:
```php
if ( isset( $block_attributes['something'], $block_attributes['style']['background']['backgroundImage']['source'] ) ) {
```

Reply:
The null coalescing operator ticket has been punted to 6.5. For 6.4, `??` is being removed. Why? A proposal has not yet been published or adopted to introduce the `??` into Core.

Reply:
If `??` is preferred for this scope of work, then I'd suggest punting this PR to 6.5 to accompany the [Trac ticket to introduce `??` into Core](https://core.trac.wordpress.org/ticket/58874).